micropyGPS.py

```diff
@@ -497,21 +497,29 @@ def gpgsv(self):
                     sat_id = int(self.gps_segments[sats])
                 except ValueError:
                     return False
+                except Exception:
```

Comment:
Can you change this to ```IndexError``` since that's the exception your commit comment references? You can even just combine them: `except (ValueError, IndexError):`